connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -211,7 +221,13 @@ public GeyserSession(GeyserConnector connector, BedrockServerSession bedrockServ
 
         this.inventoryCache.getInventories().put(0, inventory);
 
+        if (EventManager.getInstance().triggerEvent(new SessionConnectEvent(this)).isCancelled()) {
+            disconnect("");
```

Comment:
Specify reason of disconnect in event? Or at least say "Session was disconnected." (If the latter, we can probably translate it)

Reply:
I've now set a default but the plugin can override this.